.ci/job_matrix.yaml

```diff
@@ -1,46 +1,59 @@
 ---
-job: 'ucc'
+job: "ucc"
```

Comment:
Check why it changed single quotes to double - is it YAML conventions?

Reply:
What is SOF?

Reply:
stuckoverflow (https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml)